x-pack/plugins/synthetics/e2e/journeys/index.ts

```diff
@@ -5,17 +5,5 @@
  * 2.0.
  */
 
-export * from './data_view_permissions';
-export * from './read_only_user';
+export * from './uptime';
 export * from './synthetics';
```

Comment:
```suggestion
export * from './synthetics';
export * from './uptime';
```

want to run synthetics first